lib/vm.py

```diff
@@ -401,6 +401,26 @@ def get_all_efi_bins(self):
 
         return binaries
 
+    def get_vtpm(self):
+        if not self.is_uefi:
+            return None
+
+        try:
+            res = self.host.xe('vtpm-list', {'vm': self.uuid})
+            return res.splitlines()[0].split(':')[1].strip()
+        except IndexError:
+            return None
+
+    def create_vtpm(self):
+        logging.info("Creating vtpm for vm %s" % self.uuid)
+        return self.host.xe('vtpm-create', {'vm-uuid': self.uuid})
+
+    def destroy_vtpm(self):
+        vtpm_uuid = self.get_vtpm()
+        if vtpm_uuid is not None:
+            logging.info("Destroying vtpm %s" % vtpm_uuid)
+            self.host.xe('vtpm-destroy', {'uuid': vtpm_uuid}, force=True)
```

Comment:
Why is `--force` needed to destroy a vTPM?

Reply:
This is required by the `vtpm-destroy` command because of possible data loss...